tests/openadapt/privacy/providers/test_private_ai_scrub.py

```diff
@@ -0,0 +1,218 @@
+"""Module to test PrivateAIScrubbingProvider."""
+
+from io import BytesIO
+import base64
+import os
+
+from loguru import logger
+from PIL import Image
+import easyocr
+import requests
+
+from openadapt import config
+from openadapt.privacy.providers.private_ai import PrivateAIScrubbingProvider
+
+scrub = PrivateAIScrubbingProvider()
+
+try:
+    scrub.scrub_text("hello Bob smith")
+except ValueError:
+    import pytest
+
+    pytestmark = pytest.mark.skip(reason="Private AI API key not found or uin")
+
+
+def test_pdf_redaction() -> None:
+    """Test to check that the PDF redaction works."""
+    pdf_path = "tests/assets/sample_llc_1.pdf"
+    redacted_pdf_path = scrub.scrub_pdf(pdf_path)
+
+    url = "https://api.private-ai.com/deid/v3/process/files/base64"
+
+    file_type = "application/pdf"
+
+    # Read from file
+    with open(redacted_pdf_path, "rb") as b64_file:
+        file_data = base64.b64encode(b64_file.read())
+        file_data = file_data.decode("ascii")
+    os.remove(redacted_pdf_path)
+
+    payload = {
+        "file": {"data": file_data, "content_type": file_type},
+        "entity_detection": {"accuracy": "high", "return_entity": True},
+        "pdf_options": {"density": 150, "max_resolution": 2000},
+        "audio_options": {"bleep_start_padding": 0, "bleep_end_padding": 0},
+    }
+
+    headers = {
+        "Content-Type": "application/json",
+        "X-API-KEY": config.PRIVATE_AI_API_KEY,
+    }
+
+    response = requests.post(url, json=payload, headers=headers)
+    if response is None:
+        raise ValueError("Private AI request returned None")
+    response = response.json()
+    if type(response) is dict and "details" in response.keys():
+        raise ValueError(response.get("detail"))
```

Comment:
`.get` is unnecessary if we're already checking that `'details'` is in `responsel.keys()`.

Also, you should only need to do '"details" in response`

Reply:
Addressed in: [e743b6c](https://github.com/OpenAdaptAI/OpenAdapt/pull/486/commits/e743b6ce460b5213d3b4ce06717cbe9236a09aef)